contracts/utils/Address.d.ts

```diff
@@ -18,7 +22,8 @@ export const LP_KAR_KUSD: "0x0000000000000000000200000000800000000081";
 export const LP_KSM_KUSD: "0x0000000000000000000200000000820000000081";
 export const LP_LKSM_KUSD: "0x0000000000000000000200000000830000000081";
 export const Token: "0x0000000000000000000000000000000000000800";
-export const EVM: "0x0000000000000000000000000000000000000801";
-export const Oracle: "0x0000000000000000000000000000000000000802";
-export const Schedule: "0x0000000000000000000000000000000000000803";
-export const DEX: "0x0000000000000000000000000000000000000804";
+export const NFT: "0x0000000000000000000000000000000000000801";
+export const EVM: "0x0000000000000000000000000000000000000802";
+export const Oracle: "0x0000000000000000000000000000000000000803";
+export const Schedule: "0x0000000000000000000000000000000000000804";
+export const DEX: "0x0000000000000000000000000000000000000805";
```

Comment:
The pre-deploy contracts deploy at the address on the mandala testnet. so should not change it.

Reply:
can't we break it?

Reply:
I think it's better breaking it now than living with this forever

Reply:
This will take effect after resetting the mandala testnet.